jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MultiPart.java

```diff
@@ -1750,7 +1750,7 @@ private void notifyComplete()
             catch (Throwable x)
             {
                 if (LOG.isDebugEnabled())
-                    LOG.debug("failure while notifying listener {}", listener, x);
+                    LOG.atDebug().setCause(x).log("failure while notifying listener {}", listener);
```

Comment:
Not necessary here, you are in a catch block already.
The Throwable will exist.